cmd/commands/common.go

```diff
@@ -296,6 +297,16 @@ func ensureGitUserToken(ctx context.Context, opts *RuntimeInstallOptions) error
 		}
 
 		log.G(ctx).Infof("Personal git token was not provided. Using runtime git token to register user: \"%s\". You may replace your personal git token at any time from the UI in the user settings", currentUser.Name)
+
+		if opts.GitIntegrationRegistrationOpts.Username == "" {
```

Comment:
how can the GitIntegrationRegistrationOpts.Username be anything but `""` at this point? is it connected to a new flag?